source/tutorials/write-delete-documents.txt

```diff
@@ -64,12 +64,11 @@ the MongoDB Server manual.
 Delete One Document
 -------------------
 
-To delete a single document from a MongoDB collection, use the static
-``Mono.from`` method from the ``Mono`` ``Publisher`` to call the ``deleteOne()``
-method and pass in your query filter. The ``Mono`` ``Publisher`` is a
+To delete a single document from a MongoDB collection, call the ``deleteOne()``
+method and pass in your query filter. Then, pass the ``deleteOne()`` result to the static
+``Mono.from`` method from the ``Mono Publisher``. The ``Mono Publisher`` is a
```

Comment:
S: I don't have much context for Java RS, but based on the API docs wording it doesn't seem like the Mono class is referred to as the "Mono Publisher"; rather, a Mono is an implementation of Publisher. Maybe you could word as:
```suggestion
method and pass in your query filter. Then, pass the ``deleteOne()`` result to the static
``Mono.from`` method from the ``Mono`` class. The ``Mono`` class is a
```

Reply:
Applies to Delete Multiple as well

Reply:
Nora and I talked about this in person; no action